tests/test_definitions.py

```diff
@@ -66,6 +66,83 @@ def test_convert_magicc7_to_magicc6_variables(magicc7, magicc6):
     )
 
 
+@pytest.mark.parametrize("suffix", [
+    "RF",
+    "ERF"
+])
+@pytest.mark.parametrize("prefix", [
```

Comment:
```suggestion
@pytest.mark.parametrize("suffix_part_1", [
```

It's not really a prefix if it comes after the variable...